R/paginate.R

```diff
@@ -40,15 +40,26 @@
 #'   )
 req_paginate <- function(req,
                          next_request,
+                         body = NULL,
```

Comment:
I was thinking this would default to `\(resp) NULL` so that `nest_request` and `n_pages` could require a `body` argument. I think that makes the rest of the implementation quite a bit simpler.